README.md

```diff
@@ -318,7 +318,7 @@ const middleware = require('webpack-dev-middleware');
 
 // This function makes server rendering of asset references consistent with different webpack chunk/entry configurations
 function normalizeAssets(assets) {
-  if (typeof assets === 'object') {
+  if (Object.prototype.toString.call(assets) === '[object Object]') {
```

Comment:
this is not a good change, as there are many different cases which qualify an entity as an object. you should recommend use of a module like `is-plain-obj` instead

Reply:
@shellscape Yes I'm just realising this now, will change

Reply:
`isObject` from lodash is okay, but it will make a big dependency. When checking object type i usually go for `typeof assets === Object` or `assets instanceof Object`

Reply:
@ev1stensberg but the problem there is 
![image](https://user-images.githubusercontent.com/32617597/42361977-76edb124-80f1-11e8-9d91-7a39588bab35.png)

Reply:
either of those checks would catch arrays

Reply:
yeah just go with a package to make this example simple. we don't need to bikeshed over that, we just want something simple for users to consume in the docs. if they prefer another method, I'm sure they'll use it.

Reply:
👍